spec/features/ability_spec.rb

```diff
@@ -167,7 +167,7 @@
     expect(current_path).to eq(root_path)
 
     visit admin_conference_commercials_path(conference2.short_title)
-    expect(current_path).to eq(admin_conference_commercials_path(conference2.short_title))
+    expect(current_path).to eq(root_path)
```

Comment:
I guess yes, as @differentreality stated out, we allow indexing of conference Commercials! @hennevogel any idea why this fails now?

Anyway, I will investigate it!